auditlog/vegetables-service/src/main/java/io/debezium/demos/auditing/vegetables/transactioncontext/TransactionContextData.java

```diff
@@ -21,5 +21,6 @@
     @Column(name="client_date")
     public ZonedDateTime clientDate;
 
-    public String usecase;
+    @Column(name="use_date")
```

Comment:
TYPO in annotation -> use_date ? ;-)

Reply:
Uff, yes :) I could need more coffee it seems :)